app/views/idv/fail.html.slim

```diff
@@ -5,4 +5,4 @@ h1.h3.my0 = t('idv.titles.hardfail')
 p.mb1 = t('idv.messages.hardfail')
 p.mb1 = t('idv.messages.hardfail2', hours: Figaro.env.idv_attempt_window_in_hours)
 p.mb1 = t('idv.messages.hardfail3')
-p.mb1 = t('idv.messages.hardfail4')
+p.mb1 = t('idv.messages.hardfail4', sp: @sp_name)
```

Comment:
if she pastes it here, I am happy to add as part of this PR :) 

Doing two messages adds a bit of complication because you need to look at the session and see if the person was referred or not. Not too hard, but does require some ruby knowledge so probably best to collaborate between content / development on this one. 🌈